hack/config.sh

```diff
@@ -1,5 +1,5 @@
 binaries="cmd/virt-controller cmd/virt-launcher cmd/virt-handler cmd/virt-api cmd/virtctl"
-docker_images="$binaries images/haproxy images/iscsi-demo-target-tgtd images/vm-killer"
+docker_images="$binaries images/haproxy images/iscsi-demo-target-tgtd images/vm-killer images/libvirtd"
```

Comment:
I would say, create a separate issue and let us find a solution for faster deployment there.